pkg/upstream/cluster/edf.go

```diff
@@ -18,6 +18,7 @@
 package cluster
 
 import (
+	"mosn.io/mosn/pkg/configmanager"
```

Comment:
maybe we can move these const to v2 package?

Reply:
I can't make up my mind about this

Reply:
just move these to v2/upstream.go
<img width="475" alt="image" src="https://user-images.githubusercontent.com/52443884/156310382-3409bacd-03e1-473d-a892-ca700716e19d.png">

Reply:
@jizhuozhi you should fallow MOSN import code style

example:

```go
import (
        // build-in package
	"sync"
        // empty line
        // other package
	envoy_api_v2 "github.com/envoyproxy/go-control-plane/envoy/api/v2"
)
```

Reply:
> move these to pkg/config/v2, since they are common const value.
> 
> ```go
> const (
> 	MinHostWeight               = uint32(1)
> 	MaxHostWeight               = uint32(128)
> 	DefaultMaxRequestPerConn    = uint32(1024)
> 	DefaultConnBufferLimitBytes = uint32(16 * 1024)
> )
> ```

i agree with you

Reply:
Do we have another PR now?

Reply:
No, this configuration is used in more than one place, but if I submit a revised PR now, it will conflict with this PR

Reply:
There're two ways to solve this problem:
1. do all thing in this pr, it's ok, I guest that the amount of affected code won't be too much.
2. a new pr just to move those const define, and rebase the current pr to that new pr, seems little subtle.